### PR TITLE
Add abbreviator and separator on actual situation

### PR DESCRIPTION
**Pull request by Faustrox**
#### Screenshots: 
Screenshot_2020-05-29-17-54-02-865_org pathcheck covidsafepaths

### CODE REVIEW

File: app/views/DR/HomeScreen/index.js [hunk 1]
```diff
@@ -40,15 +40,35 @@ export default class HomeScreen extends Component {
...
   }
 
+  // This fuction is to abreviate or separate numbers, ex: 1000 => 1,000, 100000 => 100K
+  separateOrAbreviate = data => {
+    const { cases, deaths, recovered, todayCases } = data;
+
+    const oldCases = [cases, deaths, recovered, todayCases];
+
+    const newCases = oldCases.map(number => {
+      if (number < 1e3) return number;
```
Comment (inteljudajocu) [inline app/views/DR/HomeScreen/index.js]: Use a switch case instead of a if else chain

**Review by inteljudajocu — COMMENTED** [review]
Comments

**Review by JoseFcoEstrella — APPROVED** [review]

File: app/views/DR/HomeScreen/index.js [hunk 2]
```diff
@@ -47,14 +47,20 @@ export default class HomeScreen extends Component {
...
 
     const newCases = oldCases.map(number => {
-      if (number < 1e3) return number;
-      else if (number >= 1e3 && number < 1e5)
-        return number.toString().replace(/(\d)(?=(\d{3})+(?!\d))/g, '$1,');
-      else if (number >= 1e5 && number < 1e6)
-        return +(number / 1e3).toFixed(1) + 'K';
-      else return +(number / 1e6).toFixed(1) + 'M';
-    });
+      switch (true) {
```
Comment (inteljudajocu) [inline app/views/DR/HomeScreen/index.js]: change switch condition from true for example: to number > 0 checkings if the number is positive that way bypassing possible errors

**Review by inteljudajocu — CHANGES_REQUESTED** [review]
Comments

**Review by inteljudajocu — APPROVED** [review]

**Pull request merged**